doc/radosgw/config-fcgi.rst

```diff
@@ -572,15 +572,15 @@ To install ``swift`` client, execute the following:
 
 To test swift access, execute the following::
 
-	swift -A http://{IP ADDRESS}/auth/1.0 -U testuser:swift -K ‘{swift_secret_key}’ list
+	swift -A http://{IP ADDRESS}/auth/1.0 -U testuser:swift -K {swift_secret_key} list
```

Comment:
@weiqiaomiao i think the single quote will be read by shell, so that the special (meta) characters are not interpreted by the shell. otherwise if we have, for example, `hello$3world` in the `swift_secret_key`, the `swift` cli will get `helloworld` instead.